modules/auxiliary/gather/zookeeper_info_disclosure.rb

```diff
@@ -0,0 +1,80 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Tcp
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name' => 'Apache ZooKeeper Information Disclosure',
+      'Description' => %q{
+        Apache Zookeeper server service runs on TCP 2181 and by default, it is accessible without any authentication. This module targets Apache ZooKeeper service instances to extract information about the system environment, and application configuration.
```

Comment:
pretty long line, did you run `msftidy` or `rubocop`? `rubocop -a` may clean it up automatically.

Reply:
rubocop doesn't enforce a maximum line length for descriptions

Rubocop actually forces the `update_info` method call to be on a new line, so I've been using that as my eyeball-test for seeing whether a PR has ran rubocop or not:

```diff
  def initialize(info = {})
-    super(update_info(info,
+    super(
+      update_info(
+        info,
```

The date is also in the wrong format, the rubocop rule will autofix that too now - https://github.com/rapid7/metasploit-framework/pull/14213

For visibility - we've got a ticket to update CI to require rubocop be ran on _new_ modules, now that it seems stable enough

Reply:
@h00die msftidy was clean, have used rubocop with '-a' now. Thanks. @adfoster-r7 I have also fixed the date format now.